rules/55-zz-scsi_id.rules

```diff
@@ -0,0 +1,20 @@
+ACTION=="remove", GOTO="scsi_id_end"
```

Comment:
the naming of this file is strange, none of our other rules files uses underscores. 

maybe call this 56-fallback-scsi-id.rules? that way you don't need the weird "zz" thing, as it is still ordered before multipath anyway

Reply:
My idea was to give some "meaning" to the numbers. 55 for hardware property detection, 58 for content-related property detection, 56 and 57 for stuff that might go between. Using "56-fallback-scsi-id.rules" would violate that principle. "55-zz" looks weird at first sight, but it expresses quite well the intention of the file - being the last thing to execute before "56".
Sticking to the numbers might also make the intention more obvious to future generations of programmers.

But if you insist I can change this, no problem. After all, this is designed to be temporary (famous last words).

Reply:
yes, please, the "zz" thing would be new, and if it's temporary we really shouldn't adopt a new naming scheme for that, in particular given that temporary things tend to remain temporary for a long long time